zgallery/src/main/java/com/mzelzoghbi/zgallery/activities/ZGalleryActivity.java

```diff
@@ -84,4 +85,10 @@ public void onPageScrollStateChanged(int state) {
         hAdapter.setSelectedItem(currentPos);
         mViewPager.setCurrentItem(currentPos);
     }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        onBackPressed();
```

Comment:
Can you please make a check here for item with id android.R.id.home, so if we will be adding any feature menu items it will not conflict with them